src/locale/pt-BR.h

```diff
@@ -0,0 +1,167 @@
+#define D_STR_AIRFLOW "Fluxo de Ar"
+#define D_STR_STEP "Etapa"
+#define D_STR_NA "N/A"
+#define D_STR_INSIDE "Interno"
+#define D_STR_OUTSIDE "Externo"
+#define D_STR_LOUD "Alto"
+#define D_STR_UPPER "Mais alto"
+#define D_STR_LOWER "Mais baixo"
+#define D_STR_BREEZE "Brisa"
+#define D_STR_CIRCULATE "Circular"
+#define D_STR_CEILING "Teto"
+#define D_STR_WALL "Parede"
+#define D_STR_ROOM "Sala"
+#define D_STR_6THSENSE "Sexto sentido"
+#define D_STR_ZONEFOLLOW "Acompanhar ambiente"
+#define D_STR_FIXED "Fixo"
+
+#define D_STR_AUTO "Auto"
+#define D_STR_AUTOMATIC "Automático"
+#define D_STR_MANUAL "Manual"
+#define D_STR_COOL "Esfriar"
+#define D_STR_HEAT "Aquecer"
+#define D_STR_FAN "Ventilar"
+#define D_STR_FANONLY "Apenas ventilar"
+#define D_STR_DRY "Secar"
+#define D_STR_8C_HEAT D_STR_HEAT " 8C"
+
+#define D_STR_MAX "Max"
+#define D_STR_MAXIMUM "Máximo"
+#define D_STR_MIN "Min"
+#define D_STR_MINIMUM "Mínimo"
+#define D_STR_MED "Med"
+#define D_STR_MEDIUM "Médio"
+
+#define D_STR_HIGHEST "Mais alto"
+#define D_STR_HIGH "Alto"
+#define D_STR_HI "Médio alto"
+#define D_STR_MID "Médio"
+#define D_STR_MIDDLE "Médio baixo"
+#define D_STR_LOW "Inferior"
+#define D_STR_LO "Baixo"
+#define D_STR_LOWEST "Mais baixo"
+#define D_STR_RIGHT "Direita"
+#define D_STR_MAXRIGHT D_STR_MAX " " D_STR_RIGHT
+#define D_STR_RIGHTMAX_NOSPACE D_STR_RIGHT " (" D_STR_MAX ")"
+#define D_STR_LEFT "Esquerda"
+#define D_STR_MAXLEFT D_STR_MAX " " D_STR_LEFT
+#define D_STR_LEFTMAX_NOSPACE D_STR_LEFT " (" D_STR_MAX ")"
+#define D_STR_WIDE "Amplo"
+#define D_STR_CENTRE "Centro"
+#define D_STR_TOP "Topo"
+#define D_STR_BOTTOM "Baixo"
+
+// Separators
+#define D_CHR_TIME_SEP ':'
+#define D_STR_SPACELBRACE " ("
+#define D_STR_COMMASPACE ", "
+#define D_STR_COLONSPACE ": "
+
+#define D_STR_DAY "Dia"
+#define D_STR_DAYS D_STR_DAY "s"
+#define D_STR_HOUR "Hora"
+#define D_STR_HOURS D_STR_HOUR "s"
+#define D_STR_MINUTE "Minuto"
+#define D_STR_MINUTES D_STR_MINUTE "s"
+#define D_STR_SECOND "Segundo"
+#define D_STR_SECONDS D_STR_SECOND "s"
+#define D_STR_NOW "Agora"
+#define D_STR_THREELETTERDAYS "DomSegTerQuaQuiSexSab"
+
+#define D_STR_YES "Sim"
+#define D_STR_NO "Não"
+#define D_STR_TRUE "Verdadeiro"
+#define D_STR_FALSE "Falso"
+
+#define D_STR_REPEAT "Repetir"
+#define D_STR_CODE "Código"
+#define D_STR_BITS "Bits"
+
+// IRrecvDumpV2+
+#define D_STR_TIMESTAMP "Timestamp"
+#define D_STR_LIBRARY "Biblioteca"
+#define D_STR_MESGDESC "Descrição da mensagem"
+#define D_STR_TOLERANCE "Tolerância"
+#ifndef D_STR_IRRECVDUMP_STARTUP
+#define D_STR_IRRECVDUMP_STARTUP \
+    "IRrecvDump está rodando e aguardando por entradas IR no pino %d"
+#endif  // D_STR_IRRECVDUMP_STARTUP
+
+#ifndef D_WARN_BUFFERFULL
+#define D_WARN_BUFFERFULL \
+    "AVISO: O código IR é muito grande para o buffer (>= %d). " \
+    "Esse resultado não é confiavel e precisa ser resolvido. " \
+    "Edite e aumente o valor de `kCaptureBufferSize`."
+#endif  // D_WARN_BUFFERFULL
+
+#endif  // LOCALE_DEFAULTS_H_
```

Comment:
From the linter:
`src/locale/pt-BR.h:167:  #endif line should be "#endif  // LOCALE_PT_BR_H_"  [build/header_guard] [5]`

I'll do a commit to fix this and add a few other tests too.